staging/src/k8s.io/kubectl/testdata/apply/rc-lastapplied.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   annotations:
     kubectl.kubernetes.io/last-applied-configuration: |
       {"test":1234}
-  name: test-rc
+  name: test-last-applied
   labels:
-    name: test-rc
+    name: test-last-applied
```

Comment:
I'm hesitant to change the name of this in this shared test data file. The test that originally use this (`TestRunApplyViewLastApplied`) seems to assume that this, and the files at filenameRC and filenameRCArgs are the same object that has had different changes made to it--e.g. it is using `filenameRCLASTAPPLIED` as response data for a request for `filenameRC`. If we just need any object with a last-applied annotation, could we use `deploy-serverside.yaml`?

Reply:
Or if this was because of the fieldmanager test, the second non-annotated object could be anything, i.e. instead of `rc-managedfields-lastapplied.yaml` it could be 'rc-noexist.yaml' or even `service.yaml`?